ISISNeutronMuon/MDANSE
# Text output in the plotter

**Description of work**
To make it easier for the users to transfer their results to other software (e.g. for fitting), we add another view to the Plot Holder. It is a text viewer, which shows the analysis results as plain text.

**Fixes**
1. Improved the logger handling of exceptions and error messages.
2. Created DataWidget as an equivalent to PlotWidget.
3. Created Text as subclass of Plotter, to be used with DataWidget.

**To test**
Please load analysis results with 1D and 2D arrays (e.g. DOS and DISF). Make sure that they can be plotted.
Create a text output using the new button in plot selector.
"Plot" the results in the new tab. Check the following actions:
1. Changing the units of the axes.
5. Selecting/deselecting datasets.
6. Changing other plot properties in the dataset table on the left.
Plot properties should not affect the text output.
